basic/mobile_io.py

```diff
@@ -0,0 +1,98 @@
+import hebi
+from time import sleep, time
+
+
+class MobileIO():
+    
+    num_buttons = 8
+    num_sliders = 8
+    current_button_state = [0, 0, 0, 0, 0, 0, 0, 0]
+    current_slider_state = [0, 0, 0, 0, 0, 0, 0, 0]
+    
+    btn_states = ["", "", "", "", "", "", "", ""]
```

Comment:
These variables are class fields, not _instance_ fields (_i.e._, they are static variables). This means that multiple instances of `MobileIO` will refer to a single instance of these variables, which I assume is not the intention here.

Move these definitions into `__init__` to make them instance variables